packages/components/src/typography/sizableText.tsx

```diff
@@ -0,0 +1,19 @@
+import { styled, Text } from '@tamagui/core'
```

Comment:
Little nitpick here regarding the file name. The project follows the kebab case, so let's rename it and fix the word's spelling to `sizeable`.